.github/workflows/contract-tests.yml

```diff
@@ -41,6 +48,6 @@ jobs:
           HARDHAT_DAO_ENS_DOMAIN: 'dao.eth'
           HARDHAT_PLUGIN_ENS_DOMAIN: 'plugin.dao.eth'
           MANAGINGDAO_SUBDOMAIN: 'aragon-management'
-          MANAGINGDAO_MULTISIG_APPROVERS: '0xc8541aAE19C5069482239735AD64FAC3dCc52Ca2,0xA592E80E5E5D194ab635064e15EA35E883aB75dC'
+          MANAGINGDAO_MULTISIG_APPROVERS: '0xc8541aAE19C5069482239735AD64FAC3dCc52Ca2,0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'
```

Comment:
On localhost(hardhat network), The tess fail as I didn't put manually my deployer address.
not comfortable. first, i don't know my deployer address since it's hardhat network.

Reply:
For these tests, shouldn't we be using the prefunded wallets from Hardhat?

Reply:
ok if no address i will add the deployer to the list if also network is not mainnnet

Reply:
Better way would be: if no address present and if network hardhat or localhost add the deployer. Otherwise require that address from env is not empty